Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/eventregistration/ui/EventRegistrationTestFragment.kt

```diff
@@ -36,29 +35,6 @@ class EventRegistrationTestFragment : Fragment(R.layout.fragment_test_eventregis
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        with(binding) {
```

Comment:
Why did you remove all the TestMenu Screens? Do you see no benefits in still having them for e.g. trying something out in a quick way?

Reply:
If we remove them, we also need to remove the "Events" cards.

Reply:
Yes all removed. I kept what is needed for testing such as 
1- CheckIn matching 
2- Last locations cards

This was temporarily for testing now we have all in place ,please use the actual ones  additionally some screens were causing a conflict in the generated NavDirections